docs/endpoints.rst

```diff
@@ -80,7 +83,7 @@ To start a new endpoint run the following command::
 Starting an endpoint will perform a registration process with funcX.
 The registration process provides funcX with information regarding the endpoint.
 The endpoint establishes an outbound connection to RabbitMQ to retrieve tasks, send results, and communicate command information.
-Note that FuncX RabbitMQ outbound access is to default port 5672 and the endpoint also needs HTTPS 443.
+Note that funcX RabbitMQ outbound access is to default port 5672 and the endpoint also needs HTTPS 443.
```

Comment:
Err, that's port 5671 (amqps://).  Suggest reworking this sentence slightly however.  Perhaps:

> Note that by default, the funcX endpoint requires outbound access to the funcX services over HTTPS (port 443) and AMQPS (port 5671).

Reply:
Reworded it similarly.